### DISCUSSION TITLE
Issue after enable HTTP Server

### DISCUSSION THREAD

**Discussion by zysm0** [netbox-community/netbox#9770] [General]
Hi guys, good afternoon from Chile

i'm having the next issue after enable the HTTP server and was weird, because at the first time it works, but when i try to enter to the site it fails:

```
(venv) crozas@crozas-virtual-machine:/etc/nginx/sites-enabled$ sudo systemctl restart nginx
Job for nginx.service failed because the control process exited with error code.
```

`(venv) crozas@crozas-virtual-machine:/etc/nginx/sites-enabled$ systemctl status nginx.service | more
× nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Mon 2022-07-18 16:02:48 -04; 25s ago
       Docs: man:nginx(8)
    Process: 22636 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=1
/FAILURE)
        CPU: 9ms

jul 18 16:02:48 crozas-virtual-machine systemd[1]: Starting A high performance web server and a reverse proxy
 server...
jul 18 16:02:48 crozas-virtual-machine nginx[22636]: nginx: [emerg] unknown directive "192.168.100.80" in /et
c/nginx/sites-enabled/netbox:5
jul 18 16:02:48 crozas-virtual-machine nginx[22636]: nginx: configuration file /etc/nginx/nginx.conf test fai
led
jul 18 16:02:48 crozas-virtual-machine systemd[1]: nginx.service: Control process exited, code=exited, status
=1/FAILURE
jul 18 16:02:48 crozas-virtual-machine systemd[1]: nginx.service: Failed with result 'exit-code'.
jul 18 16:02:48 crozas-virtual-machine systemd[1]: Failed to start A high performance web server and a revers
e proxy server.
(venv) crozas@crozas-virtual-machine:/etc/nginx/sites-enabled$ 
`

Really i'm not an expert in Linux, so please your support in order to find the issue.

Thanks so much in advance

**Comment by a084ed22** [comment]
You should verify in detail what the issue is on the nginx configuration by issuing _nginx -t_
